sorts/test/bucket_sort.test.ts

```diff
@@ -0,0 +1,21 @@
+import { bucketSort } from "../bucket_sort";
+
+describe("Testing Selection sort", () => {
```

Comment:
Don't copy/duplicate test cases. You should apply the same tests to both selectionsort and bucketsort.

Reply:
It's fixed check it now.

Reply:
I'm afraid it still isn't fixed. Bucket sort and selection sort both solve the same problem, so when you're doing black-box testing, they should both get the same test cases. *In addition to that* you may do some white-box testing specific to each, but currently the test cases are duplicated.